### PR TITLE
Remove 'None' value from list cycles_has_activity in ofec_committee_h…

### PR DESCRIPTION
**Pull request by fec-jli**
## Summary (required)
C00140590 committee profile page is throwing a server error.

- Resolves #[[_3426_](https://github.com/fecgov/fec-cms/issues/3426)]
_Include a summary of proposed changes._
modify cms load_committee_history function in views.py
to clean cycles_has_activity, remove 'None' value in cycles_has_activity
committee['cycles_has_activity'] = list(filter(None, committee.get('cycles_has_activity')))

## Impacted areas of the application
committee profile page
 

## How to test
1) checkout branch
2)pytest
./manage.py test
3)npm run test-single
4) test url, compare with prod.
(wrong one)
prod: https://www.fec.gov/data/committee/C00140590/
local: http://127.0.0.1:8000/data/committee/C00140590/

(correct one should not affected)
prod: https://www.fec.gov/data/committee/C00105668/
local: http://127.0.0.1:8000/data/committee/C00105668/

We should create an separated issue to fix ofec_committee_history_mv

### CODE REVIEW

**Comment by fec-jli** [comment]
> @fec-jli if we leave the logic as a control on the cms side rather than (or in addition to) sql, then should we also filter for any non-integer value rather than only None? For expediency I am approving now.

Thanks @jason-upchurch review it. for now, we only get 'None' value from sql query return, not sure if we will get other value. I will talk to database team to see any concern. Thanks.

**Comment by jason-upchurch** [comment]
> > @fec-jli if we leave the logic as a control on the cms side rather than (or in addition to) sql, then should we also filter for any non-integer value rather than only None? For expediency I am approving now.
> 
> Thanks @jason-upchurch review it. for now, we only get 'None' value from sql query return, not sure if we will get other value. I will talk to database team to see any concern. Thanks.

Thanks @fec-jli I think the datatype in the db is specified as `integer` so I agree nothing else should come back (because we rely on the root for correct behavior).

File: fec/data/tests/test_committee.py [hunk 1]
```diff
@@ -571,11 +572,18 @@ def test_load_committee_history_uses_last_cycle_has_activity(
         assert all_candidates == []
         assert committee == first_row_data
 
+        # check if cycles_has_activity include 'None' item
+        # in function load_committee_history() in views.py (line 524)
+        # we remove 'None' value in cycles_has_activity
+        # so make sure cycles_has_activity not include 'None' value anymore,
+        # assert should is Flase.
+        assert any(x is None for x in committee['cycles_has_activity']) is False
```
Comment (lbeaufort) [inline fec/data/tests/test_committee.py]: Thanks for adding test coverage, @fec-jli! Would `assert None not in committee['cycles_has_activity']` be more clear?
